.travis.yml

```diff
@@ -1,8 +1,24 @@
 services:
 - docker
+
+before_script:
+  - docker login -u $DOCKER_USER -p $DOCKER_PASSWORD   #login to dockerhub
+  - curl https://cli-assets.heroku.com/install.sh | sh  #install heroku
+  - docker login --username=_ --password=$HEROKU_API_KEY registry.heroku.com 
+
 script:
-- docker login -u $DOCKER_USER -p $DOCKER_PASSWORD
-- docker build --target test --tag my-test-image .
-- docker run -e FLASK_APP=$FLASK_APP -e FLASK_ENV=$FLASK_ENV my-test-image todo_app/test_tasks.py
-- docker run -e TRELLO_KEY=$TRELLO_KEY -e TRELLO_TOKEN=$TRELLO_TOKEN -e TRELLO_BOARD_ID=$TRELLO_BOARD_ID -e TRELLO_TODO_LIST_ID=$TRELLO_TODO_LIST_ID -e TRELLO_DOING_LIST_ID=$TRELLO_DOING_LIST_ID -e TRELLO_DONE_LIST_ID=$TRELLO_DONE_LIST_ID my-test-image todo_app/test_client.py
-- docker run -e TRELLO_KEY=$TRELLO_KEY -e TRELLO_TOKEN=$TRELLO_TOKEN -e TRELLO_BOARD_ID=$TRELLO_BOARD_ID -e TRELLO_TODO_LIST_ID=$TRELLO_TODO_LIST_ID -e TRELLO_DOING_LIST_ID=$TRELLO_DOING_LIST_ID -e TRELLO_DONE_LIST_ID=$TRELLO_DONE_LIST_ID   my-test-image todo_app/test_system.py
+  - docker build --target test --tag my-test-image .
+  - docker run my-test-image todo_app/test_tasks.py
+  - docker run -e TRELLO_KEY=$TRELLO_KEY -e TRELLO_TOKEN=$TRELLO_TOKEN -e TRELLO_BOARD_ID=$TRELLO_BOARD_ID -e TRELLO_TODO_LIST_ID=$TRELLO_TODO_LIST_ID -e TRELLO_DOING_LIST_ID=$TRELLO_DOING_LIST_ID -e TRELLO_DONE_LIST_ID=$TRELLO_DONE_LIST_ID my-test-image todo_app/test_client.py
+  - docker run -e TRELLO_KEY=$TRELLO_KEY -e TRELLO_TOKEN=$TRELLO_TOKEN -e TRELLO_BOARD_ID=$TRELLO_BOARD_ID -e TRELLO_TODO_LIST_ID=$TRELLO_TODO_LIST_ID -e TRELLO_DOING_LIST_ID=$TRELLO_DOING_LIST_ID -e TRELLO_DONE_LIST_ID=$TRELLO_DONE_LIST_ID   my-test-image todo_app/test_system.py
+  - docker build --target prod --tag $DOCKER_USER/todo-app:latest .  
```

Comment:
You are updating the `todo-app:latest` image whenever ANY branch gets updated?

That makes the `latest` tag pretty useless.